payload/commit.go

```diff
@@ -12,22 +14,35 @@ type Commit interface {
 	SetSignature(signature []byte)
 }
 
-type commit struct {
-	signature [signatureSize]byte
-}
+type (
+	commit struct {
```

Comment:
We only use them in tests, what if we're to move them into `_test.go` files? It'd avoid importing `encoding/gob` for non-test builds.

Reply:
The case is that we use them not only in tests, we also use them as default payloads for dBFT. So it won't be possible to easily integrate dBFT with Geth node without these default payloads.

Reply:
Single-node Geth implementation used them before custom payloads implementation was added.